src/server/refresh.js

```diff
@@ -30,8 +30,6 @@ async function getAllContributorsInfo() {
     let organization = Config.organization
     let contributors = Config.contributors
 
-    interval = contributors.length // update interval
```

Comment:
@djcruz93 a better solution imo would be to make this 
`interval = contributors.length ? contributors.length : interval`